src/prommis/uky/uky_flowsheet.py

```diff
@@ -861,5 +873,956 @@ def display_results(m):
+    m.fs.R_chiller.duty = Var(initialize=13.1, units=units.MBTU / units.hr)
+    m.fs.R_chiller.duty.fix()
+    m.fs.R_chiller.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": R_chiller_accounts,
+            "scaled_param": m.fs.R_chiller.duty,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    feed_input = units.convert(
+        flowsheet.fs.leach_solid_feed.flow_mass[0]
+        + flowsheet.fs.leach_liquid_feed.flow_vol[0]
+        * flowsheet.fs.leach.liquid[0, 1].dens_mol,
+        to_units=units.ton / units.hr,
+    )
+    # Feed grade is grams of solute per 1000 grams of solution (solution ~ water since dilute concentrations)
+    feed_grade = units.convert(
+        1
+        * units.L
+        * (
+            flowsheet.fs.leach_liquid_feed.conc_mass_comp[0, "H"]
+            + flowsheet.fs.leach_liquid_feed.conc_mass_comp[0, "HSO4"]
+            + flowsheet.fs.leach_liquid_feed.conc_mass_comp[0, "SO4"]
+        ),
+        to_units=units.g,
+    )
+    m.fs.feed_input = Var(initialize=feed_input, units=units.ton / units.hr)
+    m.fs.feed_grade = Var(initialize=value(feed_grade * 1000), units=units.ppm)
+
+    hours_per_shift = 8
+    shifts_per_day = 3
+    operating_days_per_year = 336
+
+    # for convenience
+    m.fs.annual_operating_hours = Param(
+        initialize=hours_per_shift * shifts_per_day * operating_days_per_year,
+        mutable=True,
+        units=units.hours / units.a,
+    )
+
+    recovery_rate = units.convert(
+        flowsheet.fs.roaster.flow_mas_product[0], to_units=units.kg / units.hr
+    )
+    m.fs.recovery_rate_per_year = Var(
+        initialize=recovery_rate * m.fs.annual_operating_hours,
+        units=units.kg / units.yr,
+    )
+
+    # the land cost is the lease cost, or refining cost of REO produced
+    m.fs.land_cost = Expression(
+        expr=0.303736
+        * 1e-6
+        * getattr(units, "MUSD_" + CE_index_year)
+        / units.ton
+        * units.convert(m.fs.feed_input, to_units=units.ton / units.hr)
+        * hours_per_shift
+        * units.hr
+        * shifts_per_day
+        * units.day**-1
+        * operating_days_per_year
+        * units.day
+    )
+
+    solid_waste = value(
+        units.convert(
+            flowsheet.fs.leach_filter_cake.flow_mass[0], to_units=units.ton / units.hr
+        )
+    )
+
+    m.fs.solid_waste = Var(
+        m.fs.time, initialize=solid_waste, units=units.ton / units.hr
+    )  # non-hazardous solid waste
+
+    m.fs.precipitate = Var(
+        m.fs.time, initialize=0, units=units.ton / units.hr
+    )  # non-hazardous precipitate
+
+    dust = value(
+        units.convert(
+            flowsheet.fs.roaster.flow_mas_dust[0], to_units=units.ton / units.hr
```

Comment:
Same as roaster comments above.

Reply:
Same as the roaster - the property itself has this typo.